src/telemetry/session.ts

```diff
@@ -0,0 +1,24 @@
+import { readUTF8File, writeToFile } from "./../utils/file.js";
+import os from "os";
+import path from "path";
+import fs from "fs";
+import { debugLogger } from '../utils/logging.js';
+
+export async function getTelemetrySessionId(): Promise<string|undefined> {
+    const homeDirectory = os.homedir();
+    const loginConfigFilePath = path.join(homeDirectory, ".geneziotelemetryrc")
```

Comment:
Are we planning to only store the session ID in this file?

Reply:
yes